taskcluster/ci/nimbledroid/kind.yml

```diff
@@ -12,30 +12,37 @@ kind-dependencies:
     - signing
 
 
+job-defaults:
+    description: Upload APKs to Nimbledroid for performance measurement and tracking
+    run:
+        commands:
+            - [wget, {artifact-reference: '<signing/public/build/armeabi-v7a/geckoNightly/target.apk>'}, '-O', target.apk]
```

Comment:
Bug: Ah, I just noticed `geckoNightly` is just for nightly. The production build is exposed at `<signing/public/build/armeabi-v7a/geckoBeta/target.apk>`. For instance: https://firefox-ci-tc.services.mozilla.com/tasks/Q4Zi5x2xR6ygCe2WgV8feQ

So the fix should be: 
a. move this command section to `nightly` 
b. create a new section in production, like this: 

```yaml
    production:
        # ...
        run:
            commands:
                - [wget, {artifact-reference: '<signing/public/build/armeabi-v7a/geckoBeta/target.apk>'}, '-O', target.apk]
                - [python, automation/taskcluster/upload_apk_nimbledroid.py, target.apk, .nimbledroid_token]
```